controlplane/pkg/tests/utils/utils.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2019 Cisco Systems, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at:
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
+
+// Contains - check if array contains element.
+func Contains(a []string, x string) bool {
```

Comment:
please use [gomega ContainElement](https://onsi.github.io/gomega/#containelementelement-interface) instead of   `utils.Contains`.

Reply:
thnx)